src/molecules/cookie-message/CookieMessage.stories.js

```diff
@@ -0,0 +1,80 @@
+import { storiesOf } from '@storybook/vue'
+import { text } from '@storybook/addon-knobs'
+
+import ACookieMessage from './CookieMessage.vue'
+import ALink from '../../atoms/link/Link.vue'
+
+const data = {
+  components: { ACookieMessage, ALink },
+  data: () => ({
+    isVisible: true
+  }),
+  methods: {
+    close () {
+      this.isVisible = false
+    }
+  }
+}
+storiesOf('Molecules/Cookie Message', module)
+  .addParameters({ info: true })
+  .add('Default', () => ({
+    ...data,
+    props: {
+      textKnobs: {
+        default: text('Button text', 'Close')
+      },
+      linkText: {
+        default: text('Link text', 'Example link')
+      },
+      infoText: {
+        default: text('Info text', 'Find out more about their purpose and settings in your browser. By browsing the site you are agreeing to use cookies according to your browser settings.')
+      }
+    },
+    template: `
+      <a-cookie-message
+        :visible="isVisible"
```

Comment:
`visible` prop is `true` by default, so I am not sure if we need this line here and passing data with `isVisible` to component

Reply:
IMO we need this b/c we change this value after close action

```
  methods: {
    close () {
      this.isVisible = false
    }
  }
```